sparrow-py/pytests/math_test.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for the Kaskada query builder."""
+import random
+import sys
+
+import pandas as pd
+import pyarrow as pa
+import pytest
+from sparrow_py import Session
+from sparrow_py import Table
+from sparrow_py import math
+from io import StringIO
+
+
+@pytest.fixture
+def session() -> Session:
+    """Create a session for testing."""
+    session = Session()
+    return session
+
+@pytest.fixture
+def table_int64(session: Session) -> Table:
+    """Create an empty table for testing."""
+    schema = pa.schema(
+        [
+            pa.field("time", pa.string(), nullable=False),
+            pa.field("key", pa.string(), nullable=False),
+            pa.field("m", pa.int64()),
+            pa.field("n", pa.int64()),
+        ]
+    )
+    return Table(session, "table1", "time", "key", schema)
+
+def test_read_table(session, table_int64) -> None:
+    table_int64.add_csv('\n'.join([
```

Comment:
In a follow-up, I need to figure out how to cleanly write these tests -- I may look at Pandas and Polars and see what they do. Over time, we'll likely have the equivalent of our e2e tests written directly in Python.